test.sh

```diff
@@ -96,17 +93,8 @@ function test_project {
   project_dir=$1
   version=$2
   header_text "performing tests in dir $project_dir for project version v$version"
-  vendor_tarball=$tmp_root/vendor.v$version.tgz
   old_gopath=$GOPATH
```

Comment:
This should also be removed as we are not using the GOPATH any more

Reply:
Good spot 👍 tks